packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2142,8 +2152,9 @@ export class ContainerRuntime
 
 		// Do shallow copy of message, as the processing flow will modify it.
 		const messageCopy = { ...messageArg };
+		const stashedLocal = this.stashClientId === messageArg.clientId;
```

Comment:
rather than faking this, in container could we set the client id to be from the pending local state when we load?

Reply:
it seems like that is already done, so why do we need this?
0000000

Reply:
so i grabbed your branch, and it looks like if we move setting the client id before we apply the saved ops in container, it work without this.

Reply:
this seems right to be, as we need to treat remote ops created by the local client as local ops as we apply them, so stashed ops get generated correctly

Reply:
I was thinking this would require us to always save the ops even without pending ops, but I guess that's not true if we don't stash the clientId in Container when there's no pending runtime state.